pages/checkout.js

```diff
@@ -1,45 +1,53 @@
-import React from 'react';
-import { loadStripe } from '@stripe/stripe-js';
-import { Elements } from '@stripe/react-stripe-js';
+// import React from 'react';
```

Comment:
do you need these comment code lines later?

Reply:
You can remove the react lines, this page component isn't used from my point of view